cupy/cuda/nvtx.pxd

```diff
@@ -0,0 +1,53 @@
+"""
+Wrapper for NVIDIA Tools Extension Library (NVTX)
+
+"""
+cdef extern from "cupy_cuda.h":
```

Comment:
Please move this section to `nvtx.pyx`
This section is used by only `nvtx.pyx`.